css/lab-pages-editor.styl

```diff
@@ -312,12 +312,18 @@ $fontWeightBoldPlus = 700
         border-radius: $sizeXS
         padding: $sizeS $sizeM
 
+        &:has(.move-button:focus)  // Note: :has() is experimental as of Nov 2023, and not supported on Firefox
```

Comment:
It is planned to land in Firefox 121, though, so should be supported by next month.